src/vars/vars.yaml

```diff
@@ -81,11 +77,14 @@ openbare_verlichting:
   objects_endpoint: asd/services/rest_sia.php/SIAService/objecten
   types_endpoint: asd/services/rest_sia.php/SIAService/types
 trm:
-  -
-    zip: Tram KGEs.zip
+  - zip: Tram KGEs.zip
     name: tram
     shp: KGE_hartlijnen_Amsterdam_2.054.shp
-  -
-    zip: Metro KGEs.zip
+  - zip: Metro KGEs.zip
     name: metro
     shp: Metro KGEs.shp
+winkelgebieden:
+  data_schema: https://raw.githubusercontent.com/Amsterdam/schemas/master/winkelgebieden/winkelgebieden.json
```

Comment:
Deze zal nog moeten gaan verwijzen naar het gevalideerde schema op 'schemas.data.amsterdam.nl', maar, daar stond tijdens ontwikkeling nog niet de juiste definitieve versie.

Reply:
Ja eens! Ik heb voor het test werk idd de github RAW output gebruikt. Natuurlijk is het beter om https://schema.data.amsterdam.nl/datasets/winkelgebieden/winkelgebieden/ te gebruiken. Ik heb de code er op aangepast zodat dit mogelijk is.